src/main/java/com/sivalabs/ft/features/api/controllers/FavoriteFeatureController.java

```diff
@@ -0,0 +1,56 @@
+package com.sivalabs.ft.features.api.controllers;
+
+import com.sivalabs.ft.features.api.utils.SecurityUtils;
+import com.sivalabs.ft.features.domain.FavoriteFeatureService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/features/{featureId}/favorites")
```

Comment:
We are not exposing the primary keys (featureId, releaseId, etc) as part of URLs, instead we are using `code` (featureCode, releaseCode, etc) in URL patterns. So, please use {featureCode} which is `code` column in `features` table instead of {featureId}.